.travis.yml

```diff
@@ -1,16 +0,0 @@
-language: php
```

Comment:
I think we should not delete the travis file. The repo is only an example for hosters and they can/should decide what to use.